### ISSUE TITLE
`AWS::CustomerProfiles::Integration` ScheduleExpression accepts values the service rejects at create time

### ISSUE THREAD

**Issue by hakin2** [aws-cloudformation/cfn-lint#4653]
### CloudFormation Lint Version

1.55.1

### What operating system are you using?

Mac

### Describe the bug

`ScheduleExpression` on `AWS::CustomerProfiles::Integration` is modelled as `pattern: ".*"`, so cfn-lint accepts any string. Customer Profiles actually accepts only four values, compared case-insensitively:

```
rate(5minutes)
rate(15minutes)
rate(1hours)
rate(1days)
```

Anything else fails at stack create with `<value> is not a supported schedule expression`, so a template that lints clean rolls back on deploy. The trap is that the property looks like an EventBridge schedule expression but is not one: `rate(1 day)`, `rate(30minutes)` and `cron(0 6 * * ? *)` are all refused.

I confirmed the list by putting 74 candidate values through `customer-profiles put-integration`. Also rejected: every other number tried (2, 10, 30, 45 minutes; 2, 6, 12, 24 hours; 2, 7, 30 days), singular units such as `rate(1day)`, any whitespace, a leading zero (`rate(01days)`), and the `at(...)` form.

The template below exits 0 on 1.55.1 even though the first resource cannot deploy.

### Expected behavior

cfn-lint should flag a `ScheduleExpression` that Customer Profiles will refuse, and stay silent on the four accepted values in any casing.

### Reproduction template
```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  # Rejected at create time: "rate(1 day) is not a supported schedule expression"
  BadSchedule:
    Type: AWS::CustomerProfiles::Integration
    Properties:
      DomainName: example-domain
      ObjectTypeName: Customer
      FlowDefinition:
        FlowName: flow-bad
        KmsArn: arn:aws:kms:us-west-2:111111111111:key/00000000-0000-0000-0000-000000000000
        SourceFlowConfig:
          ConnectorType: S3
          SourceConnectorProperties:
            S3:
              BucketName: example-bucket
              BucketPrefix: input/
        TriggerConfig:
          TriggerType: Scheduled
          TriggerProperties:
            Scheduled:
              ScheduleExpression: "rate(1 day)"
              DataPullMode: Incremental
        Tasks:
          - TaskType: Filter
            SourceFields: [CustomerId]
            ConnectorOperator:
              S3: PROJECTION
```
  # Accepted by the service, must not be flagged
```yaml
  GoodSchedule:
    Type: AWS::CustomerProfiles::Integration
    Properties:
      DomainName: example-domain
      ObjectTypeName: Customer
      FlowDefinition:
        FlowName: flow-good
        KmsArn: arn:aws:kms:us-west-2:111111111111:key/00000000-0000-0000-0000-000000000000
        SourceFlowConfig:
          ConnectorType: S3
          SourceConnectorProperties:
            S3:
              BucketName: example-bucket
              BucketPrefix: input/
        TriggerConfig:
          TriggerType: Scheduled
          TriggerProperties:
            Scheduled:
              ScheduleExpression: "RATE(1DAYS)"
              DataPullMode: Incremental
        Tasks:
          - TaskType: Filter
            SourceFields: [CustomerId]
            ConnectorOperator:
              S3: PROJECTION
```

**Comment by hakin2** [comment]
A possible fix is a schema patch under `src/cfnlint/data/schemas/patches/extensions/all/aws_customerprofiles_integration/`:

```json
[
  {
    "op": "replace",
    "path": "/definitions/ScheduledTriggerProperties/properties/ScheduleExpression/pattern",
    "value": "(?i)^rate\\((5minutes|15minutes|1hours|1days)\\)\\Z"
  }
]
```

Applied locally, this flags `rate(1 day)` and leaves `RATE(1DAYS)` clean. I also checked it against all 74 values I tested using `regex.search`, matching how `_keywords.py` calls it: no false positives, no missed rejections.

I used `\Z` rather than `$`, because `$` matches before a trailing newline and would let `"rate(1days)\n"` through, which a YAML block scalar can produce and the API rejects.
